### PR TITLE
Minor updates for automl-tabular-classification-model evaluation notebook

### PR DESCRIPTION
**Pull request by krishr2d2**
Summary:
<br>
This PR includes the following changes:
1. Upgrades the gcpc version to the latest at the installation step.
2. Textual description corrections.
3. Adds the TargetFieldRemover component to the pipeline and changes the field names accordingly.
<br>

**REQUIRED:** Fill out the below checklists or remove if irrelevant
1. If you are opening a PR for `Official Notebooks` under the [notebooks/official](https://github.com/GoogleCloudPlatform/vertex-ai-samples/tree/main/notebooks/official) folder, follow this mandatory checklist:
- [x] Use the [notebook template](https://github.com/GoogleCloudPlatform/vertex-ai-samples/blob/main/notebooks/notebook_template.ipynb) as a starting point.
- [x] Follow the style and grammar rules outlined in the above notebook template.
- [x] Verify the notebook runs successfully in Colab since the automated tests cannot guarantee this even when it passes.
- [x] Passes all the required automated checks. You can locally test for formatting and linting with these [instructions](https://github.com/GoogleCloudPlatform/vertex-ai-samples/blob/main/CONTRIBUTING.md#code-quality-checks).
- [ ] You have consulted with a tech writer to see if tech writer review is necessary. If so, the notebook has been reviewed by a tech writer, and they have approved it.
- [x] This notebook has been added to the [CODEOWNERS](https://github.com/GoogleCloudPlatform/vertex-ai-samples/blob/main/notebooks/official/CODEOWNERS) file under the `Official Notebooks` section, pointing to the author or the author's team.
- [x] The Jupyter notebook cleans up any artifacts it has created (datasets, ML models, endpoints, etc) so as not to eat up unnecessary resources.

<br>

### CODE REVIEW

File: notebooks/official/model_evaluation/automl_tabular_classification_model_evaluation.ipynb [hunk 1]
```diff
@@ -1,1434 +1,1407 @@
 {
```
Comment (andrewferlitsch) [inline notebooks/official/model_evaluation/automl_tabular_classification_model_evaluation.ipynb:1]: <p>Tabular Classification -&gt; Tabular classification</p><p>Component -&gt; component</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/vertex-ai-samples/pull/1184/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='3'/>

Comment (andrewferlitsch) [inline notebooks/official/model_evaluation/automl_tabular_classification_model_evaluation.ipynb:1]: <p>Training Job -&gt; training job</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/vertex-ai-samples/pull/1184/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='44'/>

Comment (andrewferlitsch) [inline notebooks/official/model_evaluation/automl_tabular_classification_model_evaluation.ipynb:1]: <p>Classification -&gt; classification</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/vertex-ai-samples/pull/1184/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='47'/>

Comment (andrewferlitsch) [inline notebooks/official/model_evaluation/automl_tabular_classification_model_evaluation.ipynb:1]: <p>Run the training job</p><p>Project -&gt; project</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/vertex-ai-samples/pull/1184/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='49'/>

Comment (andrewferlitsch) [inline notebooks/official/model_evaluation/automl_tabular_classification_model_evaluation.ipynb:1]: <p>Training -&gt; training</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/vertex-ai-samples/pull/1184/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='51'/>

Comment (andrewferlitsch) [inline notebooks/official/model_evaluation/automl_tabular_classification_model_evaluation.ipynb:1]: <p>sentence case title</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/vertex-ai-samples/pull/1184/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='58'/>

Comment (andrewferlitsch) [inline notebooks/official/model_evaluation/automl_tabular_classification_model_evaluation.ipynb:1]: <p>title is not descriptive</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/vertex-ai-samples/pull/1184/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='65'/>

Comment (soheilazangeneh) [inline notebooks/official/model_evaluation/automl_tabular_classification_model_evaluation.ipynb:1]: <p>Train an* AutoML</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/vertex-ai-samples/pull/1184/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='3'/>

[the remark this answers is not in the document]
Comment (krishr2d2) [reply notebooks/official/model_evaluation/automl_tabular_classification_model_evaluation.ipynb:1]: Updated now.

Comment (krishr2d2) [reply notebooks/official/model_evaluation/automl_tabular_classification_model_evaluation.ipynb:1]: updated.

Comment (krishr2d2) [reply notebooks/official/model_evaluation/automl_tabular_classification_model_evaluation.ipynb:1]: updated.

Comment (krishr2d2) [reply notebooks/official/model_evaluation/automl_tabular_classification_model_evaluation.ipynb:1]: updated.

Comment (krishr2d2) [reply notebooks/official/model_evaluation/automl_tabular_classification_model_evaluation.ipynb:1]: updated.

Comment (krishr2d2) [reply notebooks/official/model_evaluation/automl_tabular_classification_model_evaluation.ipynb:1]: updated.

Comment (krishr2d2) [reply notebooks/official/model_evaluation/automl_tabular_classification_model_evaluation.ipynb:1]: fixed.

Comment (krishr2d2) [reply notebooks/official/model_evaluation/automl_tabular_classification_model_evaluation.ipynb:1]: updated the title.

[the remark this answers is not in the document]
Comment (krishr2d2) [reply notebooks/official/model_evaluation/automl_tabular_classification_model_evaluation.ipynb:1]: removed sentence case

[the remark this answers is not in the document]
Comment (krishr2d2) [reply notebooks/official/model_evaluation/automl_tabular_classification_model_evaluation.ipynb:1]: updated.

Comment (krishr2d2) [reply notebooks/official/model_evaluation/automl_tabular_classification_model_evaluation.ipynb:1]: added the article.

**Review by soheilazangeneh — APPROVED** [review]

**Review by andrewferlitsch — APPROVED** [review]

**Pull request merged**